internal/postgres/postgres.go

```diff
@@ -20,15 +20,25 @@ const (
 	PGDATADIR         = "/var/lib/postgresql/data/"
 )
 
-func NewPostgresContainer(image, name, postgresUsername, postgresPassword string, port int) (postgresContainer dockerservice.Container, err error) {
+type ContainerProps struct {
+	Image     string
+	Name      string
+	Username  string
+	Password  string
+	Port      int
+	Memory    string
+	CPUShares string
```

Comment:
Looks like we later convert these to integers, is there some reason we can't have them as int originally?